.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
I don't think we want these changes, or at least not all of them.  So from `ember-cli`'s perspective they have added `eslint` into the development of their own project.  They  have also put in place a very basic, default setup for each new addon/app created, just as they had with JSHint.  The difference with `eslint` though is that we already have our own configuration and rules.

Assuming we wanted to keep all of these configuration options, they should instead be moved to https://github.com/ciena-frost/eslint-config-frost-standard/blob/master/eslintrc.json and this file's contents remain as they were.  However, we don't want all of these options.  The `ocd/sort-import-declarations"` rule, for example, differs from what we already have established.   So these configurations should be evaluated and considered for inclusion in our `eslint-config-frost-standard` rules but should not just be unilaterally adopted.